va/va.h

```diff
@@ -1152,6 +1164,38 @@ typedef union _VAConfigAttribValMultipleFrame {
  */
 #define VA_ATTRIB_NOT_SUPPORTED 0x80000000
 
+/** \brief Attribute value for VAConfigAttribEncVP9Features.*/
+typedef union _VAConfigAttribEncVP9Features {
+    struct {
+        /** \brief indicates if encoder supports application forced segmentation.
+         * If so, application should provide segmentation map and segmentation parameters.
+         * Encoder will honor it without altering.max num of concurrent frames from different stream
+         */
+        uint32_t forced_segmentation_support    : 1;
+        /** \brief indicates if encoder will internally determine the segmentation map when
+         * segmentation_enabled in VAEncPictureParameterBufferVP9 is enabled and segment map surface
+         * is not provided by app.
+         */
+        uint32_t auto_segmentation_support      : 1;
+        /** \brief indicates supported segmentation features:
+         * 0b0001:  Seg QP index delta is supported;
+         * 0b0010:  Seg loop filter level delta is supported;
+         * 0b0100:  reference is supported;
+         * 0b1000:  skip is supported;
+         * Values can be combined.
+         */
+        uint32_t segment_feature_support        : 4;
+        /** \brief indicates supported encode acceleration functions:
+         * 0b0001:  Enc+Pak;
+         * 0b0010:  Hybrid Pak function;
+         * 0b0100:  Enc only function;
+        uint32_t coding_function_type           : 3;
+        /** \brief reserved bit for future, must be zero */
+        uint32_t reserved                       : 23;
+    } bits;
+    uint32_t value;
+} VAConfigAttribEncVP9Features;
```

Comment:
could we add a value to indicate the 422 or 444 support? it is not enough to use VAVP9Profile1

Reply:
Now 422/444 support is set with VAConfigAttribRTFormat value (if it set as VA_RT_FORMAT_YUV422/444).
Do we need additional field to specify this?